tests/tests-filter.cpp

```diff
@@ -23,28 +24,37 @@ TEST(filtering, store_filtered) {
     std::cout << "Configuring DicomFilter.." << std::endl;
     DicomFilter filter = DicomFilter::ParseConfig(config);
     std::cout << " configured" << std::endl;
+    size_t files_passed = 0;
+    size_t files_total = 0;
     auto test = [&](const fs::path &path){
+        char msg[1024] = {0};
+        files_total++;
         auto size = fs::file_size(path);
         std::unique_ptr<char[]> buffer(new char[size]);
-        std::cout << "Loading " << path << std::endl;
+
+        sprintf(msg,"Loading %s\n", path.c_str());
+        DEBUG_LOG(1,msg);
         std::ifstream file(path, std::ios::binary | std::ios::in);
         ASSERT_TRUE(file.is_open());
         file.read(buffer.get(),size);
         file.close();
         ASSERT_TRUE(file.good());
-        std::cout << "Parsing.." << std::endl;
         DicomFile dicom(buffer.get(), size); // valid
         ASSERT_TRUE(dicom.IsValid()); // pass
-        std::cout << "Loaded " << (dicom.IsValid() ? "valid" : "invalid") << " dicom file." << std::endl;
-        std::cout << "Filtering.." << std::endl;
+        sprintf(msg,"%s file\n",(dicom.IsValid() ? "valid" : "invalid"));
+        DEBUG_LOG(1,msg);
+
         auto [filtered_buffer,filtered_size] = filter.ApplyFilter(dicom); //memcpy ran til end of file(buffer)
         DicomFile filtered_dicom(filtered_buffer.get(),filtered_size);
         ASSERT_TRUE(filtered_dicom.IsValid()); //first sign the memcpy didn't work
-        std::cout << "Filtered data: " << (filtered_dicom.IsValid() ? "valid" : "invalid") << std::endl;
+        sprintf(msg,"Filtering: %s\n", (filtered_dicom.IsValid() ? "passed" : "failed"));
+        DEBUG_LOG(1,msg);
```

Comment:
Replacing the print lines with the new debug log macro can allow us to disable certain log lines even in unit tests. Definitely useful when debugging a unit test.